metron-platform/metron-common/src/test/java/org/apache/metron/common/stellar/StellarStatisticsFunctionsTest.java

```diff
@@ -215,10 +261,11 @@ public void testSumSquares() throws Exception {
     assertEquals(stats.getSumsq(), (Double) actual, 0.1);
   }
 
-  @Test(expected = ParseException.class)
+  @Test
   public void testKurtosisNoWindow() throws Exception {
     statsInit(0);
-    run("STATS_KURTOSIS(stats)", variables);
+    Object actual = run("STATS_KURTOSIS(stats)", variables);
+    assertEquals(stats.getKurtosis(), (Double) actual, 0.1);
   }
```

Comment:
I think you can get rid of all the "test*[No]Window" tests (for skewness, percentile, and kurtosis) and replace them with a single test just like the others (max, min, count, etc) that test both scenarios.  Those were created only for tests where there was a behavioral difference between the 'windowed' versus 'non-windowed' implementation.